app/main.py

```diff
@@ -1,25 +1,63 @@
 from abc import ABC
 
 
+class SlideLimitationValidator(ABC):
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
+
+
 class IntegerRange:
```

Comment:
The IntegerRange descriptor is supposed to work with class attributes in SlideLimitationValidator (age, height, weight), but the current implementation only sets and gets instance attributes.